assets/scss/base/_details-summary.scss

```diff
@@ -56,4 +56,18 @@
     content: "";
     display: block;
   }
-}
\ No newline at end of file
+}
+
+.summary--black {
```

Comment:
what does this summary do rather than what colour is it? could drive the name? What happens if you change the colour?